pulp_2to3_migration/app/pre_migration.py

```diff
@@ -143,6 +153,19 @@
 
             pulp2content = []
             existing_count = 0
+    if pulp2mutatedcontent:
+        # not sure we need this because of -->
+        # https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/pre_migration.py#L279
```

Comment:
That's a good question!
FWIW, the code below looks good.
I think the recent patch from Rohan might update last_unit_added for a repo and then we are all good. But we need to check it in pulp 2 for all, upload, sync and copy cases. Good catch and potential simplification! :octocat: